MapboxNavigation/NavigationMapView.swift

```diff
@@ -23,9 +23,21 @@ let sourceOptions: [MGLShapeSourceOption: Any] = [.maximumZoomLevel: 16]
 open class NavigationMapView: MGLMapView {
     
     //MARK: Class Constants
-    public static let defaultAltitude = 1000.0
-    public static let zoomedOutManeuverAltitude = 2000.0
-    public static let longManeuverDistance: CLLocationDistance = 1000.0 //meters
+    
+    /**
+     `defaultAltitude` is the altitude that the map camera initally defaults to.
```

Comment:
Nit: no need to repeat the name of the property inside its own documentation. Sometimes we do that for classes, but I guess that’s because we try to be less terse there.